.gitattributes

```diff
@@ -1,4 +1,4 @@
 go.sum linguist-generated
 examples/kubernetes/connectivity-check/connectivity-*.yaml linguist-generated
-examples/crds/*.yaml linguist-generated
```

Comment:
The commit msg has some characters leftover somehow. Trying to enter insert mode or something :monocle_face: